### ISSUE TITLE
Not allowing to generate two different short urls for same long url

### ISSUE THREAD

**Issue by abhij89** [cydrobolt/polr#441]
## Expected Behavior

It should have an option to create different short urls for same long url. Why? because multiple users can share same url to their social networks, and in order to get stats separately for both users, we need to have different short urls for same link.

## Current Behavior

It doesn't have this option. Right now it is keeping both long and short urls unique. I think there should be a flexibility to store same long urls with different short urls. Short urls should always be unique.

## Possible Solution

There should be a parameter in config or in the api itself which suggests code to allow adding same long url in db.

## Context

Like i mentioned above, two different users want to share same url to their social networks, but in order to track stats for them, we need two separate short urls for the same long url.

## Your Environment

* Environment name and version (e.g. Chrome 39, PHP 7.0, etc): Linux, PHP 5.6

**Comment by overint** [comment]
I agree that this should be a configurable option. It would likely have to be set upon installation as it is enforced by a index on the DB table.
